crates/fj-viewer/src/input/rotation.rs

```diff
@@ -26,21 +26,24 @@ impl Rotation {
 
     pub fn apply(&self, diff_x: f64, diff_y: f64, camera: &mut Camera) {
         if self.active {
-            let rotate_around =
-                self.focus_point.0.unwrap_or_else(Point::origin);
+            let rotate_around: Vector<3> =
+                self.focus_point.0.unwrap_or_else(Point::origin).coords;
```

Comment:
Why the explicit type annotation?

```suggestion
            let rotate_around =
                self.focus_point.0.unwrap_or_else(Point::origin).coords;
```